src/status_im/browser/core.cljs

```diff
@@ -311,7 +311,7 @@
 (fx/defn open-chat-from-browser
   [cofx host]
   (let [topic (string/lower-case (apply str (map filter-letters-numbers-and-replace-dot-on-dash host)))]
-    {:dispatch [:chat.ui/start-public-chat topic true]}))
+    {:dispatch [:chat.ui/start-public-chat topic {:modal? true :navigation-reset? true}]}))
```

Comment:
Is the error I mentioned, using `reset-navigation? true` throws an error if called from profile, ( we added a link to open a #status from the help section as part of this task).
Before `start-public-chat` would always call `reset-navigation? true`, and it was not configurable. I have changed the event so that it takes an `opts` parameter so we can call it with `reset-navigation? false` and no error is thrown.

Reply:
ok, got it